src/rules/speech_control.ts

```diff
@@ -1178,4 +1178,36 @@ export function initRules_bc_speech_control() {
 			}, ModuleCategory.Rules);
 		}
 	});
+
+	registerRule("speech_garble_while_talking", {
+		name: "Force garbled speech",
+		type: RuleType.Speech,
+		loggable: false,
+		shortDescription: "force PLAYER_NAME to talk as if they were gagged",
+		longDescription: "This rule forces PLAYER_NAME to talk as if they were gagged, automatically garbling all of their speech. This rule does not affect OOC or, if \"Garble whispers while gagged\" is disabled, whispers.",
```

Comment:
```suggestion
		longDescription: "This rule forces PLAYER_NAME to talk as if they were gagged, automatically garbling all of their speech. The rule does not affect OOC or whispers, unless other rules modify them.",
```
Suggested in https://discord.com/channels/842082194209112074/884526083565838367/1083826968253972631

My main concern is that failing to explicitly mention the other rule might make things a bit too vague.